functions/payment.js

```diff
@@ -98,10 +98,25 @@ exports.createPaymentIntent = functions.https.onCall(async (data, context) => {
       "venue, event or return url data missing"
     );
   }
+
+  const ticketsBoughtByUserForThisEvent = await admin
+    .firestore()
+    .collection("purchases")
+    .where("userId", "==", context.auth.token.user_id)
+    .where("eventId", "==", data.eventId)
+    .where("status", "in", ["COMPLETE"])
```

Comment:
what happens if ticket status is "PENDING" | "CONFIRMATION_FROM_STRIPE_PENDING"? Will the user be able to rebuy the ticket accidentaly?